packages/react-provider/src/hooks/use-courier-actions.ts

```diff
@@ -75,6 +75,15 @@ const useCourierActions = (state, dispatch) => {
           payload: () => events.trackEventBatch(eventType),
         });
       },
+      renewSession: async (token) => {
```

Comment:
moving this to the main courier actions so it makes sense to do window.courier.renewSession vs window.courier.inbox.renewSession